src/util/EventLogger.js

```diff
@@ -77,10 +80,10 @@ export function updateLoggerUserInfo(vue, { wallet, displayName, email }) {
   }
   if (vue.$crisp) {
     if (email) {
-      vue.$crisp.push(['set', 'user:email', [email]]);
+      vue.$crisp.push(['set', 'user:email', [email, crispToken]]);
```

Comment:
When do we need to guard get user:email exist before set?

Reply:
We do not have crispToken in the other instance, so i guard to prevent overwriting verified email with unverified email